packages/gatsby-cli/src/create-cli.js

```diff
@@ -266,7 +266,7 @@ function buildLocalCommands(cli, isLocalSite) {
 
   cli.command({
     command: `repl`,
-    desc: `Get a node repl with context of Gatsby environment, see (add docs link here)`,
+    desc: `Get a node repl with context of Gatsby environment, see (/docs/gatsby-repl/)`,
```

Comment:
```suggestion
    desc: `Get a node repl with context of Gatsby environment, see (https://www.gatsbyjs.org/docs/gatsby-repl/)`,
```

Because this one is output from the CLI it can be the full path 🙂

Reply:
Just going to commit this myself though 👍 thanks for all the work on this!

Reply:
Ahh, gotcha, thanks for pointing that out! Add that to my most recent commit.